### ISSUE TITLE
josm-plugins in beginner or editing depending on language

### ISSUE THREAD

**Issue by Nick-Tallguy** [hotosm/learnosm#413]
Please see:  https://docs.google.com/spreadsheets/d/1NlJHTYAYpyyF5o9D8UMAsSzFG7lHY2cIi2mzncx530w/edit#gid=1753852254

which shows that the section josm-plugins appears in different sections depending on the language of the viewer. This potentially leads to 404 file not found errors, and makes it difficult to manage the site. 

For instance, navigate to http://learnosm.org/en/editing/josm-plugins/ & then clicking on the link at the top of the page, which should take you to the French section, but instead give you a '404 file not found'

The file appears in 3 languages in beginner, and 6 languages in editing, so it will be much easier to move it to editing. 

The subject heading of josm-plugins appears on two different files each in a different section of the **German version**. Propose to merge them into one file, with a large 'hidden' section that can be reorganised later by someone with German language skills.

Also the beginner section is much too long, making it difficult to see the **Other guides** section - for instance, click on http://learnosm.org/en/beginner/start-josm/ & you will have to scroll to the bottom of the page before you will be able to see there is a section **Mapping with Smartphones**

This proposal can be viewed on http://nick-tallguy.github.io/en/

**Comment by Nick-Tallguy** [comment]
@michael63-osm  & @althio  I would value your opinions on this, & #414  

I'm initially aiming to reduce the beginner section and make it more relevant to beginners, whilst placing the topics in more appropriate categories. If we can reduce the number of topics in the beginner section it will make it easier to see the other categories, and hence navigate them - of course the best solution of all is a 'sliding menu' in the same way that the pencil symbol slides, so that you only see the menu when you need to instead of it always taking up half the screen.
Looking at the spreadsheet, my eventual aim is to move intermediate & advanced into other sections, so that all languages in the site have the same menus. This will make it easier to deal with translations & site maintenance in the future but it will be a long process, one stage at a time.

**Comment by michael63-osm** [comment]
Our primary goal should be consistent file (=topic) structure among the different languages. This will not only avoid 404 errors but also facilitate translation management.

As fas as JOSM plugins are concerned, the French, German and Spanish versions in the beginners' section definitively contain advanced material: they cover direct upload, mirrored download, edit gpx, print and restart in detail - topics which I did not really need so far for my own work with JOSM. Mirrored download came to my rescue only once when a task manager task contained so many data that the normal download aborted. And I certainly do not consider restarting as a frequent task when working with JOSM.

The more important plugins are mentioned where they are needed: field papers in the document on field papers, utilsplugin and building_tools in the editing section

Bottom line: I vote for removing this topic from the beginner section. We will need a document in an advanced section giving users some guidelines on recommended plugins. But I would not simply copy any existing document there without a review. Maybe a short poll about plugins to be mentioned might help here.

**Comment by Nick-Tallguy** [comment]
implemented with #431
